src/HBitcoin.Tests/FullSpvWalletTests.cs

```diff
@@ -20,24 +20,13 @@ public class FullSpvWalletTests
 		[Theory]
 		[InlineData("TestNet")]
 		[InlineData("Main")]
-		public void SycingTest(string networkString)
+		public void SyncingTest(string networkString)
 		{
-			// load wallet
-			Network network = networkString == "TestNet"? Network.TestNet:Network.Main;
-			string path = $"Wallets/Empty{network}.json";
-			const string password = "";
-			Safe safe;
-			if(File.Exists(path))
-			{
-				safe = Safe.Load(password, path);
-				Assert.Equal(safe.Network, network);
-			}
-			else
-			{
-				Mnemonic mnemonic;
-				safe = Safe.Create(out mnemonic, password, path, network);
-			}
-
+            // load wallet
+            Network network = networkString == "TestNet" ? Network.TestNet : Network.Main;
+            Safe safe = Safe.Load(string.Empty, $"../../Wallets/Empty{network}.json");
```

Comment:
Why do you have a folder with an wallet called EmptyMain.json then? Is it used anywhere? I don't see any code referring to it.